src/gamma.jl

```diff
@@ -825,7 +826,7 @@ function logabsbeta(a::T, b::T) where T<:Real
     ya, sa = logabsgamma(a)
     yb, sb = logabsgamma(b)
     yab, sab = logabsgamma(a + b)
-    (ya + yb - yab), (sa*sb*sab)
+    (ya + yb - yab), Int(sa*sb*sab)
```

Comment:
It seems another possible fix would be to ensure that `logabsgamma` always returns a sign of type `Int`. And I wonder if `logabsgamma` is type unstable as well.

Reply:
`logabsgamma` is type-stable, but it returns `Int32` since that is what the C library returns.

Reply:
We could change `logabsgamma` to return `Int`, of course.

Reply:
Update: I changed `logabsgamma` to return `Int` for the sign also.   Just for consistency (the old code was type-stable), and because `Int` is the default type so using `Int32` might lead to other problems like the one in `logbeta`.